.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
Everyone has its own local database. Typically, pg on OS X has different defaults for pg on linux, and pl may have their own db server in a raspberry π.